executor/executor.go

```diff
@@ -21,7 +23,10 @@ func (e ErrStatus) Error() string {
 }
 
 // Run executes a slice of CommandDefs
-func Run(output io.Writer, job screwdriver.JobDef) error {
+func Run(path string, output io.Writer, job screwdriver.JobDef) error {
+	if err := chdir(path); err != nil {
```

Comment:
Actually, cd in the subshell does not carry over at all right now, regardless of where we chdir or pass in a path. Each call to exec sets it to CWD for the launch process by default.